viam-cartographer.go

```diff
@@ -637,7 +639,18 @@ func (cartoSvc *cartographerService) readDataOnInterval(ctx context.Context, lid
 
 func (cartoSvc *cartographerService) getNextDataPoint(ctx context.Context, lidar lidar.Lidar, c chan int) {
 	if _, err := dim2d.GetAndSaveData(ctx, cartoSvc.dataDirectory, lidar, cartoSvc.logger); err != nil {
-		cartoSvc.logger.Warn(err)
+		if strings.Contains(err.Error(), "reached end of dataset") {
+			if !jobsDone {
+				cartoSvc.logger.Warn(err)
+				_, err = os.Create(filepath.Join(cartoSvc.dataDirectory, "Jobs_Done.txt"))
```

Comment:
[nit] Jobs_Done.txt -> job_done.txt (no 's' and lower case)

Reply:
kk I'll swap it, tbh I have been saying job's done because its a [reference to a game](https://www.youtube.com/watch?v=bBCB0y9Mgns)